fhirr4/ballerina/src/main/resources/fhirservice/http_service_builder.bal

```diff
@@ -110,10 +119,17 @@ isolated function getHttpService(Holder h, r4:ResourceAPIConfig apiConfig) retur
             }
         }
 
-        isolated resource function post [string... paths](http:Request req, http:RequestContext ctx) returns any|error {
+        isolated resource function post [string... path](http:Request req, http:RequestContext ctx) returns any|error {
 
+            string[] rawPath = regex:split(req.rawPath, "\\?");
+            string[] paths = regex:split(rawPath[0], "/");
+            _ = paths.remove(0);
+            string[] basePath = [];
+            lock {
+                basePath = name.clone();
```

Comment:
No need to assign a basePath here? can directly pass name to the getResourceMethod?